crates/bifrost/src/loglet.rs

```diff
@@ -211,7 +217,9 @@ impl std::future::Future for LogletCommit {
     ) -> Poll<Self::Output> {
         match ready!(self.rx.poll_unpin(cx)) {
             Ok(res) => Poll::Ready(res),
-            Err(_) => Poll::Ready(Err(AppendError::Shutdown(ShutdownError))),
+            Err(_) => Poll::Ready(Err(AppendError::ReconfigurationNeeded(
```

Comment:
Why is it ok to treat every `AppendError` as a `AppendError::ReconfigurationNeeded`? Shouldn't we forward `err` to give the sender control over the error type?

Reply:
We still pass AppendErrors down, this particular case catches the case if the sender of the inner oneshot channel was dropped. If has only been happening on shutdown, but recently I allowed this to happen if sequencers (either remote or local) have permanently gone away.

Reply:
Thanks for the clarification.